internal/tiltfile/docker.go

```diff
@@ -25,14 +25,17 @@ type dockerImage struct {
 	baseDockerfile     dockerfile.Dockerfile
 	configurationRef   container.RefSelector
 	deploymentRef      reference.Named
-	syncs              []pathSync
-	runs               []model.Run
-	entrypoint         string
 	cachePaths         []string
-	hotReload          bool
 	matchInEnvVars     bool
 	ignores            []string
 	onlys              []string
+	entrypoint         model.Cmd // optional: if specified, we override the image entrypoint/k8s command with this
```

Comment:
I would have called this "command" to mirror what we call it [on the ImageTarget](https://github.com/windmilleng/tilt/blob/b258a687d3a6bd1e283806d63406d59917434972/internal/model/docker_info.go#L23) / because it's most accurately overriding the k8s yaml `container.command` field, BUT that arg name is already taken by `custom_build` :-/ suggestions welcome

Reply:
ah, I would call it "command" not to make it obvious what the type is, but so it more closely matched the place in the system where we're injecting it (i.e. the `container.command` field of k8s yaml).

I think the general wisdom is that you should NOT try to name things after their types, b/c in a typesafe language like go, it should be obvious from inspecting any variable what type it is. e.g. calling something
```
var nameString string
```
is kind of redundant. (I still do it sometimes, but i'm probably not supposed to :P )